README.md

```diff
@@ -10,6 +10,17 @@ This is a simple react app to manage your ETH2 validator keystores. It allows to
 
 [See more screenshots](images)
 
+## Prerequisites
+
+This tool requires nodejs 16 or better and yarn as well as react-scripts and typescript. Installation of nodejs and npm differs by OS, for Ubuntu 20.04: 
+
+`sudo apt update && sudo apt install -y curl` and then `curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -` and finally `sudo apt install -y nodejs`.
```

Comment:
I believe there's a simpler way to install yarn (which also installs node) - see https://linuxize.com/post/how-to-install-yarn-on-ubuntu-20-04/